res/css/structures/_RoomSubList.scss

```diff
@@ -99,13 +99,13 @@ limitations under the License.
 
 .mx_RoomSubList_chevron {
     pointer-events: none;
-    background-image: url('$(res)/img/topleft-chevron.svg');
-    background-repeat: no-repeat;
+    mask: url('$(res)/img/feather-icons/dropdown-arrow.svg');
```

Comment:
hm, shouldn't we only be putting icons from feather into feather-icons?

Reply:
I wasn't sure what `feather-icons` meant at first... I guess it started as icons from https://feathericons.com/. I [asked](https://matrix.to/#/!DdJkzRliezrwpNebLk:matrix.org/$15482584132274362MGQzd:matrix.org?via=matrix.org&via=t2bot.io&via=riot.tenx.tech) this recently, and I was told it has become a place for other new icons as well, so I placed this one there.

Anyway, we can certainly move it; we just need to agree on what goes in the folder.

Reply:
Okay, we can move this one out then. When I asked @bwindels gave the impression that @nadonomy had also added other non-feather icons there, so it may take some detective work to figure out which should actually be there.

Reply:
Filed as https://github.com/vector-im/riot-web/issues/8348.

Reply:
As we've forked feather-icons with the stroke thickness changes Nad did, I wonder how useful this will be, but meh :)